run/test/specs/group_tests_add_contact.spec.ts

```diff
@@ -56,28 +58,39 @@ async function addContactToGroup(platform: SupportedPlatformsType) {
   });
   if (!addedContact && platform === 'android') {
     await device1.navigateBack(platform);
-    await device1.clickOnByAccessibilityID('Add members');
+    await device1.clickOnElementAll(new InviteContactsButton(device1));
     await device1.selectByText('Contact', userD.userName);
   }
   // Click done/apply
   await device1.clickOnByAccessibilityID('Done');
   // Click done/apply again
   await sleepFor(1000);
   await device1.clickOnElementAll(new ApplyChanges(device1));
-  // Check config message
-  await runOnlyOnIOS(platform, () =>
-    device1.waitForControlMessageToBePresent(`${userD.userName} joined the group.`)
-  );
-  await runOnlyOnAndroid(platform, () =>
-    device1.waitForControlMessageToBePresent(`You added ${userD.userName} to the group.`)
-  );
-  // Exit to conversation list
+  // Check control message
+  // "legacyGroupMemberNew": "<b>{name}</b> joined the group.",
+  const legacyGroupMemberNew = localize('legacyGroupMemberNew')
+    .withArgs({ name: userD.userName })
+    .strip()
+    .toString();
+  await device1.waitForControlMessageToBePresent(legacyGroupMemberNew as ControlMessage);
+  // await Promise.all([
+  //   device1.waitForControlMessageToBePresent(`${userD.userName} joined the group.`),
+  // device2.waitForControlMessageToBePresent(`${userD.accountID} joined the group.`),
```

Comment:
we don't want those?

Reply:
We just have alot of discrepancies between ios and android with control messages and groups. Plus i'd need to create a function that checks the accountID as a truncated number and i haven't written that yet

Reply:
So TODO comment here I guess? Then I'll resolve

Reply:
Added TODO comment